web/wp-content/themes/dctx-v2/template-parts/content-home.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-vo6hvy.php, but found phpcs-scan-vO6HvY.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).